### ISSUE TITLE
Regular Editor and Brizy

### ISSUE THREAD

**Issue by logaritym** [ThemeFuse/Brizy#260]
Hello. If I build a page with Brizy and then I click to go back to the regular Wordpress editor, switching back to Brizy erases all my progress. Please, fix this. Maybe we have some information in the regular WordPress editor which we would like to copy to Brizy or need to check something. For example, I imported my entire website from Squarespace. For every already created blog post, I clicked EDIT WITH BRIZY. Yoast SEO was showing as meta some of the texts/images from the regular posts (only for some of them, is this normal?), so I decided to take a look at it. When I switched back to Brizy, my Brizy post was gone.

P.S. Do you recommend to erase everything we have built with the regular WordPress editor before we edit with Brizy?

**Comment by tinsterman** [comment]
I agree. I was testing. Clicked on Wordpress editor and then switched back and all the design was lost.

**Comment by felipeozalmeida** [comment]
Same here. I think this should be taken as a priority by the development team. If the button "Back to Wordpress" doesn't work as expected, **hide it**. Tell users to create another page if they want the traditional TinyMCE UI again **and** offer a way for them to _export the content_. Letting users lose entire designs just because they needed to see the post content in the traditional way is nothing but a messy user experience!

**Comment by felipeozalmeida** [comment]
Alright, **I've changed the page template back to "Default"** and now things are normal **as expected**. The root cause of the problem still persists: can you tell users that, if they go back to default WordPress editor, _they'll have the page template modified_? **This wasn't obvious** and, as I am evaluating your product (along with thousand of users), I just think it'd prevent a **big** headache. This was frustrating at best.
